src/main.spec.js

```diff
@@ -97,6 +97,23 @@ test('purgeQueue deletes messages from queue', async () => {
   expect(message).toEqual(false);
 });
 
+test('default exchange', async () => {
+  const connection = await amqp.connect('some-random-uri');
+  const channel = await connection.createChannel();
+  await channel.assertQueue('retry-queue-10s');
+  await channel.assertQueue('retry-queue-20s');
+  await channel.publish('', 'retry-queue-10s', 'content-1');

```

Comment:
Should also we test `sendToQueue` method here?

Also `sendToQueue` might use `publish` to defult exchange at backend as well.
Something like

```js
{
  async publish(/*publish params*/) {
    /* publics implementation */
  },
  async sendToQueue(queueName, content, { headers } = {}) {
    this.publish(DEFAULT_EXCHANGE_NAME, routingKey, content, { headers });
  },
}
```

Reply:
great suggestion, changed